apiExamples/basic.html

```diff
@@ -0,0 +1,12 @@
+<auro-carousel centerSelected>
```

Comment:
Why does this basic example have the `centerSelected` attribute? Why is there a basic and a default example?